### PR TITLE
rename rpc h2 settings key

### PR DESCRIPTION
**Pull request by zhuangjinjin**
## What is the purpose of the change
rename rpc h2 settings key

## Brief changelog
h2.settings.header-table-size -> dubbo.rpc.tri.header-table-size
h2.settings.enable-push -> dubbo.rpc.tri.enable-push
h2.settings.max-concurrent-streams -> dubbo.rpc.tri.max-concurrent-streams
h2.setting.initial-window-size -> dubbo.rpc.tri.initial-window-size
h2.settings.max-frame-size -> dubbo.rpc.tri.max-frame-size
h2.settings.max-header-list-size -> dubbo.rpc.tri.max-header-list-size

## Verifying this change
yes

## Usage
just look at it, https://github.com/apache/dubbo/pull/8700#issuecomment-913961366

## Checklist
- [x] Make sure there is a [GitHub_issue](https://github.com/apache/dubbo/issues) field for the change (usually before you start working on it). Trivial changes like typos do not require a GitHub issue. Your pull request should address just this issue, without pulling in other changes - one PR resolves one issue.
- [ ] Each commit in the pull request should have a meaningful subject line and body.
- [ ] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
- [ ] Check if is necessary to patch to Dubbo 3 if you are work on Dubbo 2.7
- [ ] Write necessary unit-test to verify your logic correction, more mock a little better when cross module dependency exist. If the new feature or significant change is committed, please remember to add sample in [dubbo samples](https://github.com/apache/dubbo-samples) project.
- [ ] Add some description to [dubbo-website](https://github.com/apache/dubbo-website) project if you are requesting to add a feature.
- [ ] GitHub Actions works fine on your own branch.
- [ ] If this contribution is large, please follow the [Software Donation Guide](https://github.com/apache/dubbo/wiki/Software-donation-guide).

### CODE REVIEW

File: dubbo-rpc/dubbo-rpc-api/src/main/java/org/apache/dubbo/rpc/Constants.java [hunk 1]
```diff
@@ -95,10 +95,10 @@
...
-    String H2_SETTINGS_HEADER_TABLE_SIZE_KEY = "h2.settings.header-table-size";
-    String H2_SETTINGS_ENABLE_PUSH_KEY = "h2.settings.enable-push";
-    String H2_SETTINGS_MAX_CONCURRENT_STREAMS_KEY = "h2.settings.max-concurrent-streams";
-    String H2_SETTINGS_INITIAL_WINDOW_SIZE_KEY = "h2.settings.initial-window-size";
-    String H2_SETTINGS_MAX_FRAME_SIZE_KEY = "h2.settings.max-frame-size";
-    String H2_SETTINGS_MAX_HEADER_LIST_SIZE_KEY = "h2.settings.max-header-list-size";
+    String H2_SETTINGS_HEADER_TABLE_SIZE_KEY = "dubbo.rpc.tri.header-table-size";
+    String H2_SETTINGS_ENABLE_PUSH_KEY = "dubbo.rpc.tri.enable-push";
+    String H2_SETTINGS_MAX_CONCURRENT_STREAMS_KEY = "dubbo.rpc.tri.max-concurrent-streams";
+    String H2_SETTINGS_INITIAL_WINDOW_SIZE_KEY = "dubbo.rpc.tri.init-window-size";
```
Comment (EarthChen) [inline dubbo-rpc/dubbo-rpc-api/src/main/java/org/apache/dubbo/rpc/Constants.java:101]: It's better to just change the prefix . `dubbo.rpc.tri.initial-window-size`

**Review by EarthChen — CHANGES_REQUESTED** [review]

**Comment by EarthChen** [comment]
Please resolve the conflict.

File: dubbo-rpc/dubbo-rpc-api/src/main/java/org/apache/dubbo/rpc/Constants.java (hunk 1 above)
Comment (zhuangjinjin) [reply dubbo-rpc/dubbo-rpc-api/src/main/java/org/apache/dubbo/rpc/Constants.java:101]: ACK

**Comment by zhuangjinjin** [comment]
> Please resolve the conflict.

it's done

**Comment by EarthChen** [comment]
Please modify https://github.com/apache/dubbo/pull/8700#issuecomment-913961366

**Review by EarthChen — APPROVED** [review]
LGTM

**Comment by zhuangjinjin** [comment]
> Please modify [#8700 (comment)](https://github.com/apache/dubbo/pull/8700#issuecomment-913961366)

ACK

File: dubbo-rpc/dubbo-rpc-triple/src/main/java/org/apache/dubbo/rpc/protocol/tri/TripleHttp2Protocol.java [hunk 2]
```diff
@@ -55,7 +56,7 @@ public void close() {
 
     @Override
     public void configServerPipeline(ChannelPipeline pipeline, SslContext sslContext) {
-        final Configuration config = ConfigurationUtils.getGlobalConfiguration();
+        final Configuration config = ConfigurationUtils.getGlobalConfiguration(ApplicationModel.defaultModel());
```
Comment (plusmancn) [inline dubbo-rpc/dubbo-rpc-triple/src/main/java/org/apache/dubbo/rpc/protocol/tri/TripleHttp2Protocol.java:59]: Is the `defaultModel` always ok?

Comment (guohao) [reply dubbo-rpc/dubbo-rpc-triple/src/main/java/org/apache/dubbo/rpc/protocol/tri/TripleHttp2Protocol.java:59]: I will solve this  in another PR , plz just ignore here

**Review by guohao — APPROVED** [review]
LGTM

File: dubbo-rpc/dubbo-rpc-triple/src/main/java/org/apache/dubbo/rpc/protocol/tri/TripleHttp2Protocol.java (hunk 2 above)
Comment (plusmancn) [reply dubbo-rpc/dubbo-rpc-triple/src/main/java/org/apache/dubbo/rpc/protocol/tri/TripleHttp2Protocol.java:59]: LGTM

**Comment by plusmancn** [comment]
LGTM

**Pull request merged**